config/initializers/logger.rb

```diff
@@ -0,0 +1,13 @@
+# suppresses routing errors
+if Rails.env.production?
+  class ActionDispatch::DebugExceptions
```

Comment:
__[RuboCop]__ <sub>posted by Sider</sub>

Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style.